vscode/src/chat/agentic/prompts.ts

```diff
@@ -25,7 +25,7 @@ In this environment you have access to this set of tools you can use to fetch co
 [NOTES]
 1. Only use <TOOL*> tags when additional context is necessary to answer the question.
 2. You may use multiple <TOOL*> tags in a single response if needed.
-3. Never request sensitive information such as passwords or API keys.
+3. NEVER request sensitive information or files such as passwords, API keys, or .env files.
```

Comment:
```suggestion
3. NEVER request sensitive information or files such as passwords, API keys, or dotEnv files.
```

maybe this would cover more?